src/endpoints/workerskv/mod.rs

```diff
@@ -30,3 +33,7 @@ pub struct Key {
 
 impl ApiResult for Key {}
 impl ApiResult for Vec<Key> {}
+
+pub fn url_encode_key(key: &str) -> String {
```

Comment:
Is `pub` necessary here?

Reply:
great catch, I don't think so--this is only used in this module :)

Reply:
```suggestion
fn url_encode_key(key: &str) -> String {
```